### DISCUSSION TITLE
Lesson 10: import ILendingPoolAddressesProvider and DataTypes from ILendingPool.sol

### DISCUSSION THREAD

**Discussion by KentrellK** [smartcontractkit/full-blockchain-solidity-course-py#1489] [Q&A]
I am wondering why protocol-v2 is left off the path when importing ILendingPoolAddressesProvider and DataTypes? The repositories show the following paths <img width="344" alt="image" src="https://user-images.githubusercontent.com/104844401/169069444-36458e02-2366-490a-94c9-8ca38e637ac2.png">
<img width="507" alt="image" src="https://user-images.githubusercontent.com/104844401/169069605-7c5249fc-19ad-4472-abcd-20cd20d92753.png">

therefore I tried ![image](https://user-images.githubusercontent.com/104844401/169071386-80bb0a74-a470-4989-a61c-d2d0b6a97a89.png) and received the following error when compiling ![image](https://user-images.githubusercontent.com/104844401/169071661-e8f06233-3e64-4722-8afa-2cc4703ccf8d.png)

I then looked at Patrick's code and realized protocol-v2 is not in the import path . I removed it and then it compiled ![image](https://user-images.githubusercontent.com/104844401/169071995-0de75cea-786d-4d03-bbda-a2fec6ea15aa.png)

any insight on why protocol-v2 isn't in the path is appreciated.

**Comment by cromewar** [comment]
Hello @Kentrell-dev 
The reason is because when the lesson was recorded the V2 was the latest version, and now the latest is the V3
so In order to solve that your `brownie-config.yaml` should be like this:

```YAML
dependencies:
  - aave/protocol-v2@1.0.1
compiler:
  solc:
    remappings:
      - "@aave=aave/protocol-v2@1.0.1"
dotenv: .env
```
so then you can use just the regular imports:

```Solidity
import {ILendingPoolAddressesProvider} from "@aave/contracts/interfaces/ILendingPoolAddressesProvider.sol";
import {DataTypes} from "@aave/contracts/protocol/libraries/types/DataTypes.sol";
```
